sites/public/lib/helpers.tsx

```diff
@@ -78,11 +86,17 @@ export const getListings = (listings) => {
               {
                 text: getImageTagLabelFromListing(listing),
                 iconType:
-                  listing?.isVerified &&
                   listing?.marketingType === ListingMarketingTypeEnum.comingSoon
-                    ? "badgeCheck"
-                    : null,
-                iconColor: "#193154",
+                    ? "calendarBlock"
```

Comment:
I know this was already there, but can you refactor this a little so that `getImageTagLabelFromListing` is not called twice. I'm also seeing a TS warning on the tags value, that will probably be resolved by a refactor as long as getImageTagLabelFromListing returns ImageTag[] | undefined.